### PR TITLE
Have updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vighnesh192**
The repository name and folder name had to be updated.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ Before we begin, if you don't already have, download [git](https://git-scm.com/d
  - Open the terminal and navigate to the directory where you wish to clone the project using   
     `cd <Address-to-the-directory>`
  - Clone the project and navigate to working directory by typing the commands    
-`git clone https://github.com/<github_account_name>/wayback-machine-chrome.git`   
- `cd wayback-machine-chrome`
+`git clone https://github.com/<github_account_name>/wayback-machine-webextension.git`
```
Comment (anishsarangi) [inline CONTRIBUTING.md:21]: Please also change the name in the 17th line. Thank you

**Review by anishsarangi — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (vighnesh192) [reply CONTRIBUTING.md:21]: Ohh yaa..I'll do that.

Comment (vighnesh192) [reply CONTRIBUTING.md:21]: Done!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ To share either bug or features, [create an issue here](https://github.com/inter
 
 ## How to set up the development environment?
 Before we begin, if you don't already have, download [git](https://git-scm.com/downloads ) and [node.js](https://nodejs.org/en/download/ ) (You may look at relevant tutorials to set them up according to your operating system)
- - Open the [wayback machine chrome github page](https://github.com/internetarchive/wayback-machine-chrome/ ), and fork the project
+ - Open the [wayback machine chrome github page](https://github.com/internetarchive/wayback-machine-webextension/ ), and fork the project
```
Comment (anishsarangi) [inline CONTRIBUTING.md:17]: *wayback machine browser extension github page* would be better. Thanks !

**Review by anishsarangi — CHANGES_REQUESTED** [review]

**Review by anishsarangi — APPROVED** [review]

**Pull request merged**